roles/beaker/sandbox/packages/tasks/main.yml

```diff
@@ -59,3 +61,32 @@
         - python-glanceclient
         - python-novaclient
         - python-neutronclient
+
+- name: Download Firefox 52 # For selenium compatibility
+  get_url:
+    url: https://ftp.mozilla.org/pub/firefox/releases/52.7.0esr/linux-x86_64/en-US/firefox-52.7.0esr.tar.bz2
+    dest: /tmp/firefox-52.7.0esr.tar.bz2
```

Comment:
Can we introduce a check so we will not download this over and over if we have 52 in beaker sandbox already?

Reply:
This is the default behavior, as seen in docs: https://docs.ansible.com/ansible/latest/modules/get_url_module.html

Reply:
`/tmp` is the problem then

Reply:
Yes. Changed.